components/script/body.rs

```diff
@@ -14,6 +14,8 @@ use crate::dom::formdata::FormData;
 use crate::dom::globalscope::GlobalScope;
 use crate::dom::promise::Promise;
 use crate::script_runtime::JSContext;
+#[cfg(feature = "refcell_backtrace")]
+use accountable_refcell::Ref;
```

Comment:
Rather than changing all of these files like this, let's change the uses of `std::cell::Ref` to `dom::bindings::cell::Ref` and make that type re-export `std::cell::Ref` or `accountable_refcell::Ref` as needed.